src/components/HomePage/PrayerTimes.tsx

```diff
@@ -55,14 +56,28 @@ type Geo = {
   longitude?: string;
 };
 
+type HijriDateData = {
+  dayName: string;
+  month: number;
+  date: number;
+  year: number;
+};
+
 type Data = {
   geo: Geo;
   prayerTimes: PrayerTimes;
   hijriDate: string;
+  hijriDateData: HijriDateData;
```

Comment:
duplicating the key for now, 
the API also returns both keys to avoid breaking next.quran.com,
will do a follow up PR to remove the old one